src/style.scss

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  &__input {
+    box-sizing: border-box;
+    display: flex;
+    width: 100%;
+
+    margin-top: 20px;
```

Comment:
The checklist recommends being consistent with your margins. You've used `margin-top` here, but it's better to stick to either top or bottom margins to avoid potential margin collapse. This is a code style issue.